app/static/css/style.css

```diff
@@ -70,3 +69,32 @@ img {
 #error-panel #error-type {
   margin-top: 0;
 }
+
+.cursor-disabled {
+  cursor: not-allowed;
+}
+
+.cursor-none {
+  cursor: none;
+}
+
+.cursor-default {
+  cursor: default;
+}
+
+.cursor-crosshair {
+  cursor: crosshair;
+}
+
+.cursor-pointer {
+  cursor: pointer;
+}
+
+.cursor-cell {
+  cursor: cell;
+}
+
+.cursor-dot {
+  cursor: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAAAb0lEQVQ4T9WSQQ4AMQRFuZfju5cJKcG0adNduyOer3yE/5NJTlOY8yUAABGZc4hWGvUZDIiZiygRWZxhBw3qQB9ZGzis4FLpCNypeRNXDcWHQP3Dbty+VWWuzuELuzJAwEO+nHAcfWm5XNwN2z0NHyeAUAvGKOgOAAAAAElFTkSuQmCC"),
+    auto;
```

Comment:
It seems like there's a bit of an offset with the dot. Any idea how to fix that?

![cursors-offset](https://user-images.githubusercontent.com/7783288/92949692-8034c580-f429-11ea-9d8f-a31826d0b53a.gif)

Reply:
I can try to fix that offset that shouldn't be an issue, it might take some trial and error for me as I do not have a USB capture device just yet. My screen is just shows at the moment "No Signal".